lib/ardb/has_slug.rb

```diff
@@ -53,11 +53,12 @@ def reset_slug
 
       def ardb_has_slug_generate_slug
         slug_attr_value = self.send(self.class.ardb_has_slug_config[:attribute])
```

Comment:
@jcredding I'm wondering if it wouldn't be better to save off the attribute as a short var named `a` or something.  Then you wouldn't need to save off the slug attr value like this and could just call `send(a)` over and over.